content/articles/looking-at-payments.md

```diff
@@ -0,0 +1,112 @@
+Title: Looking at payments solution for SAAS in Europe
+Date: 2015-12-08
+Short_summary: Payments in Europe are complicated due to VAT, I look at what a SAAS could use
```

Comment:
summary is not a full sentence.

"Payments in Europe are complicated due to recent VAT changes. I researched several SAAS that handle the new rules." maybe?